index.html

```diff
@@ -12,8 +12,10 @@
       href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
       rel="stylesheet"
     />
+    <script async src="https://cse.google.com/cse.js?cx=554dbefc814b64487">
```

Comment:
As we discussed on Slack, you don't need to use the Google API.

Reply:
done